go.mod

```diff
@@ -39,6 +39,8 @@ require (
 	google.golang.org/protobuf v1.28.0
 )
 
+require github.com/veraison/parsectpm v0.0.0-20230419103011-ab840ff78579
```

Comment:
Same as above -- please use a verisioned release. Also, this should probly be in th `require` block above rather than its own statement.

Reply:
Done in [e456186](https://github.com/veraison/services/pull/140/commits/e456186ff16c881d428899fed3d9ee573fe03fec)